django_sirene/importers.py

```diff
@@ -1,49 +1,77 @@
 import logging
 import time
 from collections import defaultdict
-from datetime import datetime
+from datetime import datetime, timedelta
+
+from django.db.models.functions import Substr
 
-from .helpers import get_siren
 from .models import Activity, Institution, LegalStatus, Municipality
 
 logger = logging.getLogger(__name__)
 
 
-class VMAJ:
-    delete = 'E'
-    create = 'C'
-    edit_final = 'F'
-    edit_initial = 'I'
-    broadcast_out = 'O'
-    broadcast_in = 'D'
+class BaseImporter:
+    def __init__(self, rows, filename, *args, **kwargs):
+        self.filename = filename
+        self.rows = rows
+        self.log_batch_size = kwargs.get("log_batch_size", 10000)
+        self.db_batch_size = kwargs.get("db_batch_size", 100)
+        self.force = kwargs.get("force", False)
+        self.log = kwargs.get("log", False)
+
+        self.date_from = datetime.now() - timedelta(days=32)
```

Comment:
I'm not sure it is a good idea to fix that by default. We might want to change that value in case something went wrong somewhere. 🤷